src/index.js

```diff
@@ -15,18 +15,19 @@ import Auth from "layouts/Auth.js";
 import Landing from "views/Landing.js";
 import Profile from "views/Profile.js";
 
+
 ReactDOM.render(
   <BrowserRouter>
     <Switch>
       {/* add routes with layouts */}
-      <Route path="/admin" component={Admin} />
+      <Route path="/" component={Admin} />
```

Comment:
So this is fine for now. I almost wonder if we want a separate admin component, but that's a discussion for another time.